client/containers/Pub/PubSyncManager.tsx

```diff
@@ -33,7 +33,7 @@ const shimPubContextProps = {
 	updateLocalData: null as any,
 	updatePubData: null as any,
 	// @ts-expect-error ts-migrate(2554) FIXME: Expected 2-3 arguments, but got 0.
-	citationManager: new CitationManager(),
+	noteManager: new NoteManager(),
```

Comment:
You might consider passing in the same defaults as in `usePageState` here so we can remove the `@ts-expect-error` comment. Runtime-wise it's no big deal since this instance should absolutely never be touched, since consumers will get the one created in the constructor below.